### PR TITLE
Update Primary contact First Name

### PR DESCRIPTION
**Pull request by lstod**
- update owners so primary contact does not show "None" for first name if primary contact is an organization

### CODE REVIEW

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

File: bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py [hunk 1]
```diff
@@ -124,7 +124,10 @@ def _map_data(row):
 def _get_name(row):
     first_name = row.get("first_name", "")
     middle_name = row.get("middle_name", "")
-    return f"{first_name} {middle_name}".strip()
+    name = f"{first_name} {middle_name}".strip()
+    if row.get("organization_name") and name == "None":
```
Comment (mhuseinov) [inline bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py:128]: Do I get it right that based on the fix, it looks like the name "None" is an actual string from OATS DB?

**Review by mhuseinov — APPROVED** [review]
👍

**Review by trslater — APPROVED** [review]
LGTM 👍

File: bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py (hunk 1 above)
Comment (lstod) [reply bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py:128]: That's correct, which is why we were seeing the first name field be populated with "None"

Comment (lstod) [reply bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py:128]: the organization_name check is probably unnecessary but technically someone's first name could be None

**Pull request merged**